src/themes/lightTheme.ts

```diff
@@ -4,6 +4,12 @@ import i18n from '../translations/i18n';
 import { getTranslatedLabelForDisplayedRows } from '../utils/pagination';
 import { Color } from './colors';
 
+// Fonts
+import '@fontsource/barlow/400.css';
+import '@fontsource/barlow/500.css';
+import '@fontsource/barlow/700.css';
+import '@fontsource/crimson-text/400.css';
```

Comment:
har en liten følelse av at disse egentlig helst burde vært i `index.js`, men det vil virke her også, så lenge vi bruker lightTheme. fra docs: `Then within your app entry file or site component, import it in.` 
hva tror du?

Reply:
Hm. Vil tro de burde importeres et hakk tilbake ja. Vi får ikke tilgang på fontene hvis vi ikke bruker lighttheme?

Reply:
hvis lightTheme aldri importeres vil jeg ikke tro det nei

Reply:
Flytte importene til index da eller?